e2e/run.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+# color codes
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+
+function getPassboltSecret() {
+    local apiVersion=$1
+    kubectl get \
+        passboltsecrets.${apiVersion}.passbolt.tagesspiegel.de \
```

Comment:
```suggestion
        "passboltsecrets.${apiVersion}.passbolt.tagesspiegel.de" \
```

Reply:
We are executing a command against the K8s api. Bash completion would never suggest that you reference it with quotes.

Reply:
If I'm not mistaken the script, as is, would break, if there is a space in `apiVersion`, it should be quoted to make sure the variable expansion works as expected and the args aren't cut up

Reply:
Spaces are not allowed in the API selection, so the expected result is actually a "crash". If the input is wrong, the result will also be wrong.

Reply:
that's not good behaviour, you should implement a check then and fail gracefully, as you can't predict how it would fail otherwise